src/Downloader.php

```diff
@@ -79,7 +79,6 @@ public function setRepositories(array $repos)
      */
     public function getRecipes(array $operations): array
     {
-        $max = 1000;
```

Comment:
I would prefer to keep the var and replace the hardcoded `1000` in the code instead.